### PR TITLE
Create WP dashboard widget to show daily uniques.

### PR DESCRIPTION
**Pull request by sashadoes**
## Summary

Addresses issue:

- #631 

## PR Author Checklist

- [ ] My code is tested and passes existing unit tests.
- [ ] My code has an appropriate set of unit tests which all pass.
- [x] My code is backward-compatible with WordPress 5.2 and PHP 5.6.
- [x] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [ ] My code has proper inline documentation.
- [x] I have added a QA Brief on the issue linked above.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Code Reviewer Checklist

- [x] Run the code.
- [x] Ensure the acceptance criteria are satisfied.
- [x] Reassess the implementation with the IB.
- [x] Ensure no unrelated changes are included.
- [ ] Ensure CI checks pass.
- [x] Check Storybook where applicable.
- [x] Ensure there is a QA Brief.

## Merge Reviewer Checklist

- [x] Ensure the PR has the correct target branch.
- [x] Double-check that the PR is okay to be merged.
- [x] Ensure the corresponding issue has a ZenHub release assigned.
- [x] Add a changelog message to the issue.

### CODE REVIEW

**Comment by sashadoes** [comment]
Quite not sure why VRT tests fail. I got the same on develop branch

File: assets/js/components/wp-dashboard/WPDashboardUniqueVisitorsChartWidget.js [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+			[ __( 'Unique Visitors', 'google-site-kit' ) ],
+			[ ( x ) => parseFloat( x ).toLocaleString() ]
+		) || [];
+
+	const dates = googleChartData.slice( 1 ).map( ( [ date ] ) => date );
+	const options = {
+		...WPDashboardUniqueVisitorsChartWidget.chartOptions,
+		hAxis: {
+			...WPDashboardUniqueVisitorsChartWidget.chartOptions.hAxis,
+			ticks: dates,
```
Comment (techanvil) [inline assets/js/components/wp-dashboard/WPDashboardUniqueVisitorsChartWidget.js:124]: The IB specifies that the x-axis should only show the first and last dates for the axis labels.

In order to achieve this, set `ticks` to be an array containing only the first and last date, and set `chartArea.left` and `chartArea.left` to `20` (or maybe a touch less) in order to provide space for the labels to render.

File: assets/js/components/wp-dashboard/WPDashboardUniqueVisitorsChartWidget.js [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+	const isZeroChart = ! googleChartData
+		.slice( 1 )
+		.some(
+			( datum ) =>
+				datum[ currentValueIndex ] > 0 ||
+				datum[ previousValueIndex ] > 0
+		);
+
+	if ( isZeroChart ) {
+		options.vAxis.viewWindow.max = 100;
```
Comment (techanvil) [inline assets/js/components/wp-dashboard/WPDashboardUniqueVisitorsChartWidget.js:154]: This line should not be necessary if we are not showing the y-axis labels.

File: assets/js/components/wp-dashboard/WPDashboardUniqueVisitorsChartWidget.js [hunk 3]
```diff
@@ -0,0 +1,242 @@
...
+		options.hAxis.ticks = [ new Date() ];
+	} else {
+		options.vAxis.viewWindow.max = undefined;
```
Comment (techanvil) [inline assets/js/components/wp-dashboard/WPDashboardUniqueVisitorsChartWidget.js:157]: This line should not be necessary if we are not showing the y-axis labels.

File: assets/js/components/wp-dashboard/WPDashboardUniqueVisitorsChartWidget.js [hunk 4]
```diff
@@ -0,0 +1,242 @@
...
+		},
+	},
+	vAxis: {
+		gridlines: {
+			color: '#eee',
+		},
+		minorGridlines: {
+			color: '#eee',
+		},
+		textStyle: {
```
Comment (techanvil) [inline assets/js/components/wp-dashboard/WPDashboardUniqueVisitorsChartWidget.js:216]: These `textStyle` and `titleTextStyle` options should not be needed if not showing the y-axis labels or title.

**Review by techanvil — CHANGES_REQUESTED** [review]

File: assets/js/components/wp-dashboard/WPDashboardUniqueVisitorsChartWidget.js [hunk 5]
```diff
@@ -0,0 +1,242 @@
...
+import WidgetReportError from '../../googlesitekit/widgets/components/WidgetReportError';
+import GoogleChart from '../GoogleChart';
+const { useSelect, useInViewSelect } = Data;
+
+const WPDashboardUniqueVisitorsChartWidget = () => {
+	const analyticsModule = useSelect( ( select ) =>
+		select( CORE_MODULES ).getModule( 'analytics' )
+	);
+	const analyticsModuleActive = analyticsModule?.active;
+	const analyticsModuleConnected = analyticsModule?.connected;
```
[the remark this answers is not in the document]
Comment (techanvil) [reply assets/js/components/wp-dashboard/WPDashboardUniqueVisitorsChartWidget.js:44]: Update: Actually, I have just realised that `isModuleConnected` does a check for both the active and connected state of the module, so you should not need to use `isModuleActive` here, only `isModuleConnected`.

https://github.com/google/site-kit-wp/blob/a1327c4f3c572a326d51fc6fe5e7c695e2859f98/assets/js/googlesitekit/modules/datastore/modules.js#L1099

**Comment by sashadoes** [comment]
Thank you @techanvil for your review and feedback. 
I made an update.

File: assets/js/components/wp-dashboard/WPDashboardUniqueVisitorsChartWidget.js [hunk 6]
```diff
@@ -0,0 +1,215 @@
...
+		gridlines: {
+			color: '#fff',
+		},
+		textStyle: {
+			color: '#616161',
+			fontSize: 12,
+		},
+	},
+	vAxis: {
+		textPosition: 'none',
```
Comment (techanvil) [inline assets/js/components/wp-dashboard/WPDashboardUniqueVisitorsChartWidget.js:197]: Please restore the `gridlines.color` setting as this is still needed for the horizontal grid lines.

```js
		gridlines: {
			color: '#eee',
		},
```

**Review by techanvil — CHANGES_REQUESTED** [review]
Thanks @sashadoes! Just one small change and I think this should be good to go.

**Pull request merged**